storage/mysql/log_storage.go

```diff
@@ -88,12 +88,15 @@ var (
 )
 
 type mySQLLogStorage struct {
+	admin storage.AdminStorage
 	*mySQLTreeStorage
```

Comment:
(style?) structs normally seem to have anonymous fields before named fields, although I've not seen a style recommendation about it anywhere